app/lib/features/chat/widgets/receipts_builder.dart

```diff
@@ -1,18 +1,114 @@
 import 'package:acter/common/providers/chat_providers.dart';
 import 'package:acter/common/themes/app_theme.dart';
 import 'package:acter_avatar/acter_avatar.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show Convo, EventSendState;
+import 'package:atlas_icons/atlas_icons.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:quds_popup_menu/quds_popup_menu.dart';
+import 'package:flutter_chat_types/flutter_chat_types.dart' as types;
 
 class ReceiptsBuilder extends ConsumerWidget {
```

Comment:
So the ReceiptsBuilder is more than just about who has seen the message now? Then how about renaming it into something more appropriate? Like ... `MessageMetadataBuilder` or something?